src/router.js

```diff
@@ -1,37 +1,40 @@
 export default function (app) {
   return {
-    model: match(app.model, location.pathname),
+    state: {},
```

Comment:
Is this why you were hesitant on setting a default value for `var state //= app.state || []`? 

On line **2** of `app.js`